services/frontend/src/scenes/People/PeopleLayout.js

```diff
@@ -1,15 +1,52 @@
 import React from "react";
 import "./People.scss";
+import {ProfileCard} from "components/";
+import classNames from "classnames";
+import {ProjectCard} from "../Projects/ProjectsLayout";
 
-const PeopleLayout = () => {
-    return (
-        <div className="people">
-            <div className="people-jumbotron">
-                <h1 className="people-tagline">WELCOME TO THE SKILLHUB!</h1>
-                <button className="people-create-People">CREATE YOUR ACCOUNT</button>
-            </div>
-        </div>
-    );
+const PeopleLayout = ({profiles, activeFilter, onFilterClick}) => (
+    <div className={"people"}>
+        <PeopleHeader
+            activeFilter={activeFilter}
+            onFilterClick={onFilterClick}
+        />
+        <PeopleList people={sortProfiles(profiles)} />
+    </div>
+);
+
+const sortProfiles = (profiles) => profiles.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
Styling: This should be declared before `PeopleLayout`, since it's used _in_ `PeopleLayout`.